MetabolicAtlas/MetabolicAtlas
# feat: debounce kcat filter

**Related issue(s) and PR(s)**  
This PR closes #1055. 

Only behavior of kcat filter in changed, the other GotEnzymes filters a built in Vue-Good-Table and seems to be debounced by default. 

**Type of change**  

 
**List of changes made**  

Added debounce to kcat min and max filter inputs.

**Screenshot of the fix**  


**Testing**  


1. Go to http://localhost/gotenzymes/reaction/R01677 (for example) 
2. Open developer tools and go to Network tab. 
3. Click "step up" in min value input a couple of times as fast as you can 
4. Verify that a request is only sent for the last value you entered (should be something like `http://localhost/api/v2/gotenzymes/enzymes?filters[reaction_id]=R01677&filters[kcat_values]={"min":4}&pagination[page]=1&pagination[pageSize]=50&pagination[column]=gene&pagination[isAscending]=true`
5. Decrease the value slowly and make sure there is a request passed for every step (and that the data is updated according to the filter) 
6. Type number in the input field and verify that is working too
7. Repeat 1-6 for max value input. 


**Further comments**  


**Definition of Done checklist**
